### PR TITLE
Multilingual support

### PR DESCRIPTION
**Pull request by pmviva**
This implements multilingual support in I think a better way, without some hacks that I made in the previous pull request.

I made some changes in the way you hardcoded some links, specially links to "/blog" and the link back to previous section, so they were easy to add the language prefix.

Also note that there's a bug in Hugo regarding RSSLinks:
I changed the RSSLink from {{ .Site.RSSLink }} to {{ .Site.RSSLink | relLangURL }} but Hugo doesn't take the language into consideration, it always gets the feed for the default language.

I hope I didn't miss anything and didn't introduce any bugs.

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -27,14 +27,19 @@ <h3 class="subtitle is-3 {{ if .Site.Params.fadeIn | default true }}fade-in two{
     <div class="hero-foot {{ if .Site.Params.fadeIn | default true }}fade-in three{{ end }}">
     <hr>
     <nav class="nav-center">
-        <a class="nav-item" href="#about">About</a>
+        <a class="nav-item" href='{{ "/#about" | relLangURL }}'>{{ i18n "ABOUT_LINK" . }}</a>
```
Comment (bep) [inline layouts/index.html]: From my recollection, I think this should be  "#about" to make it work on sites with a sub-path in baseURL.

File: i18n/en.toml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+[blog]
+  other = "Blog"
+[projects]
+  other = "Projects"
+[404_TITLE]
```
Comment (bep) [inline i18n/en.toml]: This is a matter of taste, but doing it same for all would be good. I would prefer "404_title" (i.e. lowercase)

File: layouts/index.html [hunk 3]
```diff
@@ -92,17 +99,19 @@ <h2 class="title is-2 has-text-centered">About</h2>
...
     <div class="container has-text-centered">
-    <h2 class="title is-2">Contact</h2>
+    <h2 class="title is-2">{{ i18n "CONTACT_TITLE" . }}</h2>
         <div class="markdown">
-            {{ range .Data.Pages }}
-                {{if eq .Title "Contact" }}
-                        {{.Content}}
-                    {{end}}
-                {{ end }}
+          {{ range .Data.Pages }}
```
Comment (bep) [inline layouts/index.html]: I would recommend having a look at `.Site.GetPage` etc. to improve this. Also see the headless bundles in Hugo if all you want is compose the home page with some content sections.

File: README.md [hunk 4]
```diff
@@ -43,11 +43,24 @@ Start with:
...
+$ hugo new about.md
 $ hugo new contact.md
 ```
 Then edit the markdown files with the content you'd like shown in your main page's About and Contact sections.
 
+Edit the about page's front matter to have:
+
+```yaml
+about_page: true
+```
```
Comment (bep) [inline README.md]: See my comments below. Same for the contact_page.

File: exampleSite/content/about.es.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (bep) [inline exampleSite/content/about.es.md]: Hugo tip: you can now put each language in its own content folder and drop the ".es" naming. Might not be a good idea for smaller sites.

File: layouts/index.html (hunk 1 above)
Comment (pmviva) [reply layouts/index.html]: Thanks, i will run some tests regarding this and check.

File: i18n/en.toml (hunk 2 above)
Comment (pmviva) [reply i18n/en.toml]: Yeah, its a matter of taste but it's good to have coherence in naming of keys, i'll update them all to lowercase.

File: exampleSite/content/about.es.md (hunk 5 above)
Comment (pmviva) [reply exampleSite/content/about.es.md]: Hi @bep , is this anywere documented? I haven't found any of this in the multilingual mode documentation of Hugo: https://gohugo.io/content-management/multilingual/

Comment (bep) [reply exampleSite/content/about.es.md]: Hmm... Seems I have a back log of stuff to document. http://gohugo.io/news/0.38-relnotes/ Also see the Gohugoio source for an example.

Comment (hivickylai) [reply exampleSite/content/about.es.md]: Thank you @bep!!

And thank you @pmviva for your extensive work and patience :bowing_woman:

**Comment by Hanzei** [comment]
@pmviva @vickylai Is there anything I can do to help push this feature?

**Comment by Hanzei** [comment]
@pmviva Would you mind if I use your code to finish this feature?

**Comment by pmviva** [comment]
@Hanzei I don't mind you use my code to finish the functionality. I haven't had much time lately to spend on this...

**Pull request closed without merging**